imports/plugins/core/ui/client/components/button/buttonSelect.js

```diff
@@ -29,13 +29,13 @@ class ButtonSelect extends Component {
   }
 
   handleDefaultState = () => {
-    const props = this.props;
+    const { props } = this;
     let defaultButton = props.buttons.filter(button => {
       if (button.active === true) {
         return button;
       }
     });
-    defaultButton = defaultButton[0];
+    [defaultButton] = defaultButton;
```

Comment:
This code should actually use `props.buttons.find` instead. Not sure if you want to fix that now.